src/index.html

```diff
@@ -7,6 +7,357 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="MOYO logo"
+        class="logo__logo"
```

Comment:
Don't use same names for blocks and elements.
And also, we don't need separate block for logo, style it as element of `header`

Reply:
Fixed this one as well. Made img a part of the header.

Max width issue is also fixed.